tests/test_form.py

```diff
@@ -128,14 +132,22 @@ def test_get_metadata(self):
 
     def test_normalize_metadata(self):
         """Avoid some type of strings."""
+        webcompat.app.config['EXTRA_LABELS'] = ['dance', 'powder']
         cases = [('blue sky -->', 'blue sky'),
                  ('blue sky ---->>', 'blue sky'),
                  ('', ''),
                  ('blue sky ', 'blue sky'),
                  ('bad_bird <script>', ''),
                  ('bad_bird <script-->>', ''),
                  ('a' * 300, ''),
-                 (None, None)
+                 (None, None),
+                 (['dance', 'powder'], 'dance, powder'),
+                 (['dance ', 'powder'], 'dance, powder'),
+                 ([' dance -->', 'powder -->'], 'dance, powder'),
+                 (['dance<script>', 'powder -->'], 'powder'),
+                 (['dance', None], 'dance'),
+                 (['dance<script>', ''], ''),
+                 (['not', 'allowed'], '')
```

Comment:
with the modifications proposed you can remove these additional tests, or add the ones which you think are necessary.

Reply:
Let's just remove them, they're not important now.